src/index.ts

```diff
@@ -202,13 +202,17 @@ function setInterfaceElements (
     const propType = prop.getType().getText()
 
     // Determine if the type is an object
-    const propTypeIsObject = !isNativeType(propType)
+    let propTypeIsObject = !isNativeType(propType)
 
     // If type is an object change label
     const isArray = propTypeIsObject && propType.includes('[]')
+    const isEnum = prop.getType().isEnum()
+    if (isEnum) {
+      propTypeIsObject = false
+    }
     const propLabel = propTypeIsObject
       ? `Object${isArray ? '[]' : ''}`
-      : getCapitalized(propType)
+      : (isEnum ? 'Enum' : getCapitalized(propType))
```

Comment:
It's probably time to refactor this into helper functions or a dictionary of label getters by type

```ts
function getPropLabel (propType: string): string {
  const propTypeIsEnum = prop.getType().isEnum()
  const propTypeIsObject = !propTypeIsEnum && !isNativeType(propType)
  const propTypeIsArray = propTypeIsObject && propType.includes('[]')

  if (propTypeIsArray) return 'Object[]'
  if (propTypeIsObject) return 'Object'
  if (propTypeIsEnum) return 'Enum'
  
  return getCapitalized(propType)
}
```